app/middleware/stripe-setup/check-vat-number-not-submitted.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+// Local dependencies
+const { ConnectorClient } = require('../../services/clients/connector_client')
+const connector = new ConnectorClient(process.env.CONNECTOR_URL)
+const { renderErrorView } = require('../../utils/response')
+const paths = require('../../paths')
+
+module.exports = function checkVatNumberNotSubmitted (req, res, next) {
+  if (!req.account) {
+    renderErrorView(req, res, 'Internal server error')
```

Comment:
Seems weird to use the message "Internal server error" rather than the default "There is a problem with the payments platform". But this is what the other middlewares do so 🤷‍♂️

Reply:
yeah probably something we can visit when we do selfservice improvements